src/Mod/Path/PathTests/TestPathSlot.py

```diff
@@ -0,0 +1,264 @@
+        box0.Height = 10.0
+        box1 = doc.addObject('Part::Box', 'Box')
+        box1.Length = 5.0  # X
+        box1.Width = 15.0  # Y
+        box1.Height = 5.0  # Z
+        box1.Placement = FreeCAD.Placement(FreeCAD.Vector(5.0, 0.0, 5.0), FreeCAD.Rotation(FreeCAD.Vector(0,0,1), 0))
+        doc.recompute()
+        cut = doc.addObject('Part::Cut', 'Cut')
+        cut.Base = box0
+        cut.Tool = box1
+        doc.recompute()
+
+        # Create Job object
+        PathJob.Create('Job', [doc.Cut], None)
+        doc.recompute()
+
+    @classmethod
+    def setUpClass(cls):
+        """setUpClass()...
+        This method is called upon instantiation of this test class.  Add code and objects here
+        that are needed for the duration of the test() methods in this class.
+        This method does not have access to the class `self` reference.  This method
+        is able to call static methods within this same class.
+        """
+        # FreeCAD.Console.PrintMessage("TestPathSlot.setUpClass()\n")
+        cls._setup_test_environment()
+
+    @classmethod
+    def tearDownClass(cls):
+        """tearDownClass()...
+        This method is called prior to destruction of this test class.  Add code and objects here
+        that cleanup the test environment after the test() methods in this class have been executed.
+        This method does not have access to the class `self` reference.  This method
+        is able to call static methods within this same class.
+        """
+        # FreeCAD.Console.PrintMessage("TestPathSlot.tearDownClass()\n")
+        FreeCAD.closeDocument(FreeCAD.ActiveDocument.Name)
+
+    def setUp(self):
+        """setUp()...
+        This method is called prior to each test() method.  Add code and objects here
+        that are needed for multiple test() methods.
+        """
+        self.doc = FreeCAD.ActiveDocument
+        self.con = FreeCAD.Console
+
+    def tearDown(self):
+        """tearDown()...
+        This method is called after each test() method. Add cleanup instructions here.
+        Such cleanup instructions will likely undo those in the setUp() method.
+        """
+        pass
+
+    def _set_depths_and_heights(self, op):
+        # Set Depths
+        # set start and final depth in order to eliminate effects of stock (and its default values)
+        op.setExpression('StartDepth', None)
+        op.StartDepth.Value = 10.0
+        op.setExpression('FinalDepth', None)
+        op.FinalDepth.Value = 5.0
+        op.setExpression('StepDown', None)
+        op.StepDown.Value = 5.0
+
+        # Set Heights
+        # default values used
+
+    def _format_point(self, cmd):
+        """Accepts command dict and returns point string coordinate"""
+        x = round(cmd["X"], 2)
+        y = round(cmd["Y"], 2)
+        return "({}, {})".format(x, y)
+
+    def test00(self):
+        '''Test Slot on horizontal rectangular face: ReverseDirection=True; LayerMode=Single-pass'''
```

Comment:
It is good practice to include the expected outcome of a test - the description should tell me what the test intends to verify.

Reply:
Okay.  Thanks.  I will edit the `test##()` method docStrings throughout all the unit tests I am creating to better communicate the purpose of the test and desired outcome.